lib/membrane/rtcp/parser.ex

```diff
@@ -72,6 +72,10 @@ defmodule Membrane.RTCP.Parser do
 
   defp process_rtcp(%RTCP.SenderReportPacket{ssrc: ssrc} = packet, metadata) do
     event = %RTCPEvent{
+      # TODO: remove empty array assignment of reports
+      # reports get assigned to empty array
+      # to avoid copying it when sending to other processes, they are currently not used
+      # anywhere  but once they are the reassignment should be removed
       rtcp: %{packet | reports: []},
```

Comment:
I'd remove that anyway, that shouldn't impact performance much while it impacts readability a lot